articles/golang-advanced-programming-concepts/index.md

```diff
@@ -0,0 +1,174 @@
+
+*Go is a powerful statically-typed, open-source, and procedural programming language that is rapidly gaining popularity in the IT industry*. In the [first article](https://www.section.io/engineering-education/golang-part-1-introduction/), we looked at the history of Go, its purpose, and its installation, and in the [second article](https://www.section.io/engineering-education/golang-part-2-programming-basics/), we explored basic programming concepts such as variables, input/output, etc. In this article, we are going to go over advanced programming concepts such as functions, structures, and arrays. 
+
```

Comment:
You could add a Prerequisites section that defines the pre-requisites required for this article. I know that you have mentioned your articles part 1 and part 2, but a separate section that specifies the same would be more elegant.